### PR TITLE
People view

### PR DESCRIPTION
**Pull request by mko02**
**Added a People tab**
Screenshot 2024-10-03 at 11 46 18 PM  

**Display of "all" people in the class.** 
I will update the fetch to include pagination so we can actually fetch all users in a later PR.  
Screenshot 2024-10-03 at 11 45 42 PM

**Display of all classes the user has with a student**. 
I will add the name of the student in a later PR.
Screenshot 2024-10-03 at 11 45 24 PM

### CODE REVIEW

File: CanvasPlusPlayground.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -417,7 +429,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"CanvasPlusPlayground/Preview Content\"";
-				DEVELOPMENT_TEAM = 6MTCHYNPAT;
+				DEVELOPMENT_TEAM = YN82V53CTG;
```
Comment (rahulon12) [inline CanvasPlusPlayground.xcodeproj/project.pbxproj:432]: Can you unstage these changes?

File: CanvasPlusPlayground/PeopleManager.swift [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        guard let (data, _) = await CanvasService.shared.fetch(.getPeople(courseId: courseID, bookmark: "")) else {
+            print("Failed to fetch files.")
+            return [];
+        }
+        
+        do {
+            let enrollments = try JSONDecoder().decode([Enrollment].self, from: data)
+            return enrollments
+            
+            // TODO: continue to query for next page and append to our lists
```
Comment (rahulon12) [inline CanvasPlusPlayground/PeopleManager.swift:44]: Can you create an issue for this?

**Review by rahulon12 — APPROVED** [review]
LGTM, some nits

File: CanvasPlusPlayground.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mko02) [reply CanvasPlusPlayground.xcodeproj/project.pbxproj]: fixed

File: CanvasPlusPlayground/PeopleManager.swift (hunk 2 above)
Comment (mko02) [reply CanvasPlusPlayground/PeopleManager.swift:44]: Created

**Pull request merged**